invenio_jsonschemas/config.py

```diff
@@ -33,8 +33,27 @@
 JSONSCHEMAS_URL_SCHEME = 'https'
 """Default url scheme for schemas."""
 
-JSONSCHEMAS_REPLACE_REFS = False
-"""Whether to resolve $ref before serving a schema."""
-
 JSONSCHEMAS_LOADER_CLS = None
 """Loader class used in ``JSONRef`` when replacing ``$ref``."""
+
+JSONSCHEMAS_TRANSFORM = []
+"""List of JSONSchema transformations to be used. ex: ['refs', 'allOf']"""
+
+JSONSCHEMAS_TRANSFORMATIONS = {
+    'allOf': 'invenio_jsonschemas.transform:transform_all_of',
+    'refs': 'invenio_jsonschemas.transform:transform_refs'
+}
+"""List of available JSONSchema transformations"""
+
+JSONSCHEMAS_REST_ENDPOINTS = dict()
```

Comment:
As said IRL we can do without this config variable and just list the query parameters in the URL.